pkg/cache/cache.go

```diff
@@ -368,6 +368,10 @@ func (c *Cache) ClusterQueueTerminating(name string) bool {
 	return c.clusterQueueInStatus(name, terminating)
 }
 
+func (c *Cache) ClusterQueuePending(name string) bool {
```

Comment:
Probably just using ClusterQueueActive is enough?

Reply:
Yes, that's right.

Reply:
Done.